### ISSUE TITLE
`out.width` and `out.height` warning as ignored for pptx output format

### ISSUE THREAD

**Issue by cderv** [yihui/knitr#2003]
Among docx, pptx, rtf, and odt, pptx is the only format that does not support width and height.

Do we need warning when output is pptx and out.width and out.height are specified?

_Originally posted by @atusy in https://github.com/yihui/knitr/issues/1746#issuecomment-523700310_

Following https://github.com/yihui/knitr/pull/1746 we never added the warning in knitr. Pandoc still does not support `width` and `height` for PPTX output (https://github.com/jgm/pandoc/issues/4586) 

It would be useful to add a warning about this when output format is detected to be pptx. 

cc @apreshill

**Comment by yihui** [comment]
Yes, we can add a warning. My only small concern is that the size specification might be supported in a future version of Pandoc. At that time, we will need to condition the warning on the Pandoc version.

**Comment by yihui** [comment]
This is an upstream limitation in Pandoc, which does not support setting the image width/height for `pptx` output (https://github.com/jgm/pandoc/issues/4586). It needs to be fixed there rather than in knitr. Closing as there's nothing actionable on our side; if Pandoc adds support, `out.width`/`out.height` will start working for `pptx` automatically.
